osidb/models.py

```diff
@@ -832,6 +833,34 @@ def _validate_notabug_flaw_affected(self):
                 f"{affected.ps_component} is affected by a flaw solved as NOTABUG.",
             )
 
+    def _validate_incomplete_errata_comment(self):
+        """
+        Alerts in case comments are missing affected product(s) for a released erratum.
+
+        Due to [bug 659234](https://bugzilla.redhat.com/show_bug.cgi?id=659234),
+        Errata Tool would leave Bugzilla comments informing of a released erratum without a
+        product name. This issue has been fixed but such comments may still appear in older
+        flaws.
+        """
+        empty_erratum_comment = re.compile(r"This issue has been addressed.*\n{4}Via")
+
+        incomplete = []
+        for comment in self.comments.all():
+            if "is_private" in comment.meta_attr and ast.literal_eval(
+                comment.meta_attr["is_private"]
+            ):
+                continue
+
+            if empty_erratum_comment.match(comment.text):
+                incomplete.append("#" + str(comment.external_system_id))
```

Comment:
Also regarding my comment about the hstore key search, this might be also refactored to something like:
```python
empty_erratum_comment_pattern = r"This issue has been addressed.*\n{4}Via"

for comment in self.comments.filter(meta_attr__is_private=True, text__iregex=empty_erratum_comment_pattern):
    incomplete.append("#" + str(comment.external_system_id))
```

Reply:
makes sense will fix both.